packages/run-protocol/src/collect.js

```diff
@@ -7,8 +7,10 @@ export const mapValues = (obj, f) =>
   harden(fromEntries(entries(obj).map(([p, v]) => [p, f(v)])));
 
 /** @type { <X, Y>(xs: X[], ys: Y[]) => [X, Y][]} */
-export const zip = (xs, ys) => harden(xs.map((x, i) => [x, ys[i]]));
+export const zip = (xs, ys) => harden(xs.map((x, i) => [x, ys[+i]]));
```

Comment:
Is there a note or issue somewhere that this file should be moved to a more generic place so its exports can be reused?

Reply:
I added a note in the restival doc